syft/generic/pointers/pointer_tensor.py

```diff
@@ -305,8 +299,7 @@ def remote_send(
         message = TensorCommandMessage.communication(
             self.id_at_location, self.location.id, [destination.id], kwargs
         )
-        self.owner.send_msg(message=message, location=self.location)
-        return self
+        return self.owner.send_msg(message=message, location=self.location)
```

Comment:
Why are we changing that?

Reply:
Because remote_get is a remote operation which shouldn't change the object you're working on; it's still the same pointer

Reply:
Oh no we're doing the opposite!

Reply:
Will fix this